alpha/apps/kaltura/lib/model/objectfilters/EntryServerNodeFilter.class.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.filters
+ */
+class EntryServerNodeFilter extends baseObjectFilter {
+
+	/* (non-PHPdoc)
+	 * @see myBaseObject::init()
+	 */
+	public function init()
+	{
+		$this->fields = kArray::makeAssociativeDefaultValue(array(
+			'_gte_created_at',
```

Comment:
Don't think we will need to search by created_at.
Is this any scenario that will require this?